add cole and kloss files (#3)
vtieto/instruments11-080221: cole.md

Richie Cole 

> In 1969, he joined drummer Buddy Rich's Big Band. After working with Lionel Hampton's Big Band and Doc Severinsen's Big Band, he formed his own quintet and toured worldwide, developing his own "alto madness" bebop style in the 1970s and early 1980s. He formed the Alto Madness Orchestra in the 1990s.[4]

Reviewer: This is one way to leave a comment.

> In 1969, he joined drummer Buddy Rich's Big Band. After working with Lionel Hampton's Big Band and Doc Severinsen's Big Band, he formed his own quintet and toured worldwide, developing his own "alto madness" bebop style in the 1970s and early 1980s. He formed the Alto Madness Orchestra in the 1990s.[4]

Reviewer: ```suggestion
In 1969, he joined drummer Buddy Rich's Swinging Big Band. After working with Lionel Hampton's Big Band and Doc Severinsen's Big Band, he formed his own quintet and toured worldwide, developing his own "alto madness" bebop style in the 1970s and early 1980s. He formed the Alto Madness Orchestra in the 1990s.[4]
```

> Cole performed and recorded with Eddie Jefferson,[4] Nancy Wilson, Tom Waits, The Manhattan Transfer, Hank Crawford, Freddie Hubbard, Eric Kloss, Bobby Enriquez, Phil Woods,[4] Sonny Stitt, Art Pepper, and Boots Randolph.[4] He recorded over fifty albums, including his albums Hollywood Madness (Muse, 1979) and Richie Cole Plays West Side Story (Music Masters, 1997), a tribute to Leonard Bernstein.

Reviewer: ```suggestion
Cole performed and recorded with Eddie Jefferson,[4] Nancy Wilson, Tom Waits, The Manhattan Transfer, Hank Crawford, Freddie Hubbard, Eric Kloss, Bobby Enriquez, Phil Woods,[4] Sonny Stitt, Art Pepper, and Boots Randolph.[4] He recorded over sixty albums, including his albums Hollywood Madness (Muse, 1979) and Richie Cole Plays West Side Story (Music Masters, 1997), a tribute to Leonard Bernstein.
```

> Cole performed and recorded with Eddie Jefferson,[4] Nancy Wilson, Tom Waits, The Manhattan Transfer, Hank Crawford, Freddie Hubbard, Eric Kloss, Bobby Enriquez, Phil Woods,[4] Sonny Stitt, Art Pepper, and Boots Randolph.[4] He recorded over fifty albums, including his albums Hollywood Madness (Muse, 1979) and Richie Cole Plays West Side Story (Music Masters, 1997), a tribute to Leonard Bernstein.

Reviewer: More detail here ... ?

> Cole performed and recorded with Eddie Jefferson,[4] Nancy Wilson, Tom Waits, The Manhattan Transfer, Hank Crawford, Freddie Hubbard, Eric Kloss, Bobby Enriquez, Phil Woods,[4] Sonny Stitt, Art Pepper, and Boots Randolph.[4] He recorded over fifty albums, including his albums Hollywood Madness (Muse, 1979) and Richie Cole Plays West Side Story (Music Masters, 1997), a tribute to Leonard Bernstein.

Reviewer: ```suggestion
Cole performed and recorded with Eddie Jefferson,[4] Nancy Wilson, Tom Waits, The Manhattan Transfer, Hank Crawford, Freddie Hubbard, Eric Kloss, Bobby Enriquez, Phil Woods,[4] Sonny Stitt, Art Pepper, and Boots Randolph.[4] He recorded over fifty albums, including his albums Hollywood Madness (Muse, 1979) and Richie Cole Plays West Side Story (Music Masters, 1997), a tribute to Leonard Bernstein. He also recorded "New York Afternoon."
```

> He was appointed to the Board of the National Jazz Service Organization and the Board for the National Endowment for the Arts where he served as chairman for one year. He was a charter member of the International Association of Jazz Educators.

Reviewer: This is a comment for multiple lines.

> He was appointed to the Board of the National Jazz Service Organization and the Board for the National Endowment for the Arts where he served as chairman for one year. He was a charter member of the International Association of Jazz Educators.

Reviewer: ```suggestion
He was appointed to the Board of the National Jazz Service Organization and the Board for the National Endowment for the Arts where he served as chairman for two years. He was a charter member of the International Association of Jazz Educators.
```
